run-cnf-suites.sh

```diff
@@ -66,5 +66,12 @@ if [[ ! -f "/proc/1/cgroup" ]] || grep -q init\.scope /proc/1/cgroup; then
 	cd ..
 fi
 
+if [[ -z "${TNF_PARTNER_SRC_DIR}" ]]; then
```

Comment:
This breaks github workflow. Maybe the github workflow should align with how developers typically do this? i.e. checkout partner repo and run this script with the env var set, instead of invoking the github action from the other repo directly?

Reply:
@jun-rh @edcdavid I fixed this problem, I added this env var to pre-main.yaml.

Reply:
@test-network-function/cnfcert Team, please comment if you have better suggestion than this mandatory env var.

Reply:
For now, I am ok with making this (partner pods installation) run in the container case only, so that the customer case is still covered and no extra step introduced for developers.